### PR TITLE
Chapter 4:  Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by xuankienmg**

### CODE REVIEW

**Comment by xuankienmg** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-    <title>Sample_App</title>
+    <title><%= full_title(yield(:title)) %></title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+    <%= stylesheet_link_tag    'application', media: 'all',
+      'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application',
+      'data-turbolinks-track': 'reload' %>
```
Comment (phamvanbk11) [inline app/views/layouts/application.html.erb]: dùng nháy "

**Comment by phamvanbk11** [comment]
commented

**Comment by xuankienmg** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**